Controls/Controls.js

```diff
@@ -829,7 +833,8 @@ define([
             control._value = settings.value || '';
             control._disabled = settings.disabled || false;
             control._title = settings.title || '';
-
+            control._autoBlankState = !!settings.autoBlankOption || settings.autoBlankOptionName;
```

Comment:
It feels a bit strange to me to set the autoBlankState to true if the autoBlankOptionName (setting) has been set.